### ISSUE TITLE
TieBreaker value of query depends on params exucution order

### ISSUE THREAD

**Issue by alikhil** [olivere/elastic#1343]
### Which version of Elastic are you using?

[ ] elastic.v7 (for Elasticsearch 7.x)
[X] elastic.v6 (for Elasticsearch 6.x)
[ ] elastic.v5 (for Elasticsearch 5.x)
[ ] elastic.v3 (for Elasticsearch 2.x)
[ ] elastic.v2 (for Elasticsearch 1.x)

### Please describe the expected behavior

`tie_breaker` should be set right, it should be `0.2`, like this:

```json
{
  "multi_match": {
    "fields": [
      "title^9"
    ],
    "fuzziness": "AUTO",
    "minimum_should_match": "70%",
    "query": "iphone",
    "slop": 50,
    "tie_breaker": 0.2,
    "type": "most_fields"
  }
}
```

### Please describe the actual behavior

But it set to `1`:

```json
{
  "multi_match": {
    "fields": [
      "title^9"
    ],
    "fuzziness": "AUTO",
    "minimum_should_match": "70%",
    "query": "iphone",
    "slop": 50,
    "tie_breaker": 1,
    "type": "most_fields"
  }
}
```

### Any steps to reproduce the behavior?

```golang

package main

import (
	"encoding/json"
	"log"

	"github.com/olivere/elastic"
)

func main() {
	var queryString = "iphone"

	var q = elastic.NewMultiMatchQuery(queryString,
		"title^9",
	).
		MinimumShouldMatch("70%").
		Fuzziness("AUTO").
		Slop(50).
		TieBreaker(0.2).
		Type("most_fields")

	var s, err = q.Source()
	if err != nil {
		log.Fatal(err)
	}
	data, err := json.MarshalIndent(s, "", "  ")
	if err != nil {
		panic(err)
	}
	log.Printf("%v", string(data))

}

```

But if you reorder it like this:
```golang
		
	query.Slop(50).
		Type("most_fields").
		TieBreaker(0.2)
```

`tie_breaker` will be set to `1`

**Comment by olivere** [comment]
Thanks for reporting. Yeah, that's a problem. It is caused by me initializing the values in the same way defined as the [Java source](https://github.com/elastic/elasticsearch/blob/7.7/server/src/main/java/org/elasticsearch/index/query/MultiMatchQueryBuilder.java#L107-L141). However, I agree that this is probably not what you want on the Go side of things, i.e. when setting the `Type` you most probably don't want it to have [the caller's value for `TieBreaker` being reset](https://github.com/olivere/elastic/blob/release-branch.v7/search_queries_multi_match.go#L64-L89).

I think this should be fixed in a way that this library should always send the specified values over the wire, regardless of whether they make sense or not.

I'll mark this as a bug and fix in the next release.

**Comment by olivere** [comment]
Will be fixed in 6.2.33 and 7.0.17.
